tools/detect-unused-images.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DOCS_PROJECT_PATH=$SCRIPT_DIR/..
+
+# we need to move it to make find print relative paths
+pushd "$DOCS_PROJECT_PATH/modules" > /dev/null
+readarray -d '' modules < <(find . -mindepth 1 -maxdepth 1 -type d -print0)
+
+unused_images=""
+for module in "${modules[@]}"
+do
+    pushd "$module" > /dev/null
+
+    if [ ! -d "./images" ]; then        
+        # This module does not images"
+        popd > /dev/null
+        continue
+    fi
+
+    readarray -d '' images < <(find "images" -name '*.png' -print0)
+    for image in "${images[@]}"
+    do
+        #`../` instead of `images/` is used in the documentation references
+        image=${image#"images/"}
+        image_with_che_context="${image/che/{project-context\}}"
+        image_with_crw_context="${image/crw/{project-context\}}"
```

Comment:
I'm not sure that master should contain crw-related images.
Currently there is `keycloak/crw-keycloak-username-readonly-theme.png`

I can rework it in way so each branch defines project context value.

Reply:
There should be nothing mentioning crw here... Historical mistakes. It will help for the cleanup.

Reply:
Let me then update the PR